package.json

```diff
@@ -46,7 +46,7 @@
     "vscode-debugadapter": "^1.36.0",
     "vscode-debugprotocol": "^1.36.0",
     "vscode-extension-telemetry": "^0.1.2",
-    "vscode-languageclient": "~5.2.1",
+    "vscode-languageclient": "~6.0.1",
```

Comment:
Do you want to consider changing the ~ here to ^?

Reply:
Done. I wasn't sure if there was some reason that was a ~.

Reply:
There was a time when a new feature got added which wasnt supported yet, caused some problems, so we moved to getting only patch updates.